projects/plugins/boost/app/features/image-guide/src/ui/Bubble.svelte

```diff
@@ -0,0 +1,93 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+	import { backOut } from 'svelte/easing';
+	import { fly } from 'svelte/transition';
+
+	export let oversizedBy: number;
+	export let index: number;
+
+	const severity = oversizedBy > 4 ? 'high' : oversizedBy > 2 ? 'medium' : 'normal';
+
+	let mounted = false;
+	onMount( () => ( mounted = true ) );
+	const scaleConfig = {
+		delay: 150 + 50 * index,
+		duration: 250,
+		y: 2,
+		easing: backOut,
+	};
+
+	$: oversizedLabel =
+		oversizedBy < 9 ? oversizedBy.toFixed( 1 ) : `${ Math.floor( oversizedBy ) }+`;
+</script>
+
+{#if mounted}
```

Comment:
🤔 What does this check achieve? If it's not mounted none of this stuff will be rendered anyway, right?

Reply:
I think I was too lazy to find another solution at the time to prevent the bubbles from flashing and triggering the fly-in animation correctly. I need to double-check if that's still an issue, or maybe just clean it up a bit.